Sources/StoreHelper/Core/PrePurchaseSubscriptionInfo.swift

```diff
@@ -56,21 +85,18 @@ public struct PrePurchaseSubscriptionInfo: Hashable {
     /// single element with the promo id (which will be nil) and price/period for the offer.
     /// - If there are no offers, the value of this property will be an array with a single element with the standard price/period.
     public var purchasePriceForDisplay: [PurchasePriceForDisplay]? {
-        if promotionalOffersEligible, let promotionalOffers, promotionalOffers.count > 0 {
+        if promotionalOffersEligible, !promotionalOffers.isEmpty {
```

Comment:
Using `isEmpty` is better than using `count` to tell if an array has elements

Reply:
I checked and yes you're correct! Apple recommend using isEmpty!